Jenkinsfile

```diff
@@ -324,6 +341,47 @@ pipeline {
           // FIXME can't be defined globally due to a bug in Jenkins:
           // https://issues.jenkins-ci.org/browse/JENKINS-49826
           matrixBuilder = { def matrix, int level ->
+            variations.failFast = true
+            variations["slurm"] = {
+              try {
+                node("KNL_PRIO") {
+                  parallel "allocation": {
+                    try {
+                      timeout(time: 2, unit: 'HOURS') {
+                        // Allocate a new job
+                        sh """
+                          salloc --job-name=mardyn-test --nodes=1-2 \
```

Comment:
more than two nodes possible.

Reply:
See above. I will set it to 1-3, we can't do more in interactive mode, unfortunately.